reference/resources/bgpconfig.md

```diff
@@ -65,14 +65,14 @@ spec:
 
 | Field       | Description                 | Accepted Values   | Schema | 
 |-------------|-----------------------------|-------------------|--------|
-| name | Name or identifier for the community. | | string |
+| name | Name or identifier for the community. This should be used in [prefixAdvertisements](#prefixAdvertisements) to advertise the community value. | | string |
```

Comment:
I think it's worth wording this part a bit more explicitly that nothing takes effect unless `prefixAdvertisements` is used.